addOns/plugnhack/src/main/java/org/zaproxy/zap/extension/plugnhack/brk/ClientBreakpointsUiManagerInterface.java

```diff
@@ -19,9 +19,9 @@
  */
 package org.zaproxy.zap.extension.plugnhack.brk;
 
-import org.zaproxy.zap.extension.brk.BreakpointMessageInterface;
-import org.zaproxy.zap.extension.brk.BreakpointsUiManagerInterface;
-import org.zaproxy.zap.extension.brk.ExtensionBreak;
+import org.zaproxy.addon.brk.BreakpointMessageInterface;
+import org.zaproxy.addon.brk.BreakpointsUiManagerInterface;
```

Comment:
should these classes be also added in the list of allowed class names? @thc202 For eg: 
```
 register("org.zaproxy.addon.brk.ExtensionBreak") {
                classnames {
                    allowed.set(listOf("org.zaproxy.addon.brk.ExtensionBreak","org.zaproxy.addon.brk.BreakpointMessageInterface",
,"org.zaproxy.addon.brk.BreakpointsUiManagerInterface"))
                }
                dependencies {
                    addOns {
                        register("requester") {
                            version.set("7.*")
                        }
                    }
                }
            }
        }
        ```